src/main/java/oshi/util/FileUtil.java

```diff
@@ -21,10 +22,24 @@
  * @author widdis[at]gmail[dot]com
  */
 public class FileUtil {
+
 	public static List<String> readFile(String filename) throws IOException {
-		File f = new File(filename);
-		byte[] bytes = Files.readAllBytes(f.toPath());
-		String s = new String(bytes, "UTF-8");
-		return Arrays.asList(s.split(System.getProperty("line.separator")));
+		List<String> result = new ArrayList<String>();
+		BufferedReader br = null;
+		try {
+			br = new BufferedReader(new FileReader(filename));
+			for (;;) {
+				String line = br.readLine();
+				if (line == null)
+					break;
+				result.add(line);
+			}
+		} catch (FileNotFoundException e) {
```

Comment:
That was residual from implementing one step at a time, and catching that before later deciding to just throw IOExceptions.